--- ./scripts/mysql_install_db.sh.orig	2024-10-30 09:38:15.000000000 -0500
+++ ./scripts/mysql_install_db.sh	2024-11-09 13:56:45.234179859 -0600
@@ -653,13 +653,14 @@
 if test "$cross_bootstrap" -eq 0 && test -z "$srcdir"
 then
   s_echo
-  s_echo "To start mariadbd at boot time you have to copy"
-  s_echo "support-files/mariadb.service to the right place for your system"
+  s_echo "To start mariadbd at boot time make sure that /etc/rc.d/rc.mysqld"
+  s_echo "is executable:"
+  s_echo
+  s_echo "chmod +x /etc/rc.d/rc.mysqld"
 
   if test "$auth_root_authentication_method" = normal
   then
     echo
-    echo
     echo "PLEASE REMEMBER TO SET A PASSWORD FOR THE MariaDB root USER !"
     echo "To do so, start the server, then issue the following command:"
     echo
@@ -670,7 +671,6 @@
     echo "strongly recommended for production servers."
   else
     echo
-    echo
     echo "Two all-privilege accounts were created."
     echo "One is root@localhost, it has no password, but you need to"
     echo "be system 'root' user to connect. Use, for example, sudo mysql"
