--- ./doc/Makefile.in.orig	2024-10-29 07:52:23.000000000 -0500
+++ ./doc/Makefile.in	2024-10-29 13:58:30.321184358 -0500
@@ -521,7 +521,7 @@
 	        dirmngr.texi scdaemon.texi tools.texi wks.texi \
                 gpg-card.texi
 
-myman_pages = gpg.1 gpgv.1 gpgsm.1 gpg-agent.1 dirmngr.8 scdaemon.1 \
+myman_pages = gpg2.1 gpgv2.1 gpgsm.1 gpg-agent.1 dirmngr.8 scdaemon.1 \
                 watchgnupg.1 gpgconf.1 addgnupghome.8 gpg-preset-passphrase.1 \
 		gpg-connect-agent.1 gpgparsemail.1 gpgtar.1 gpg-mail-tube.1 \
 		applygnupgdefaults.8 gpg-wks-client.1 gpg-wks-server.1 \
